### PR TITLE
All Assignments till Ruby TDD

### PR DESCRIPTION
**Pull request by ashwinvidiyala**
Feedback is appreciated!

### CODE REVIEW

File: ashwin_vidiyala/Rails/layouts_assignment/app/views/posts/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  <input type="hidden" name="authenticity_token" value="<%= form_authenticity_token %>">
+
+  <label for="title">Title</label><br>
+  <input type="text" name="post[title]"><br><br>
+
+  <label for="content">Content</label><br>
+  <textarea name="post[content]" rows="8" cols="80"></textarea><br><br>
+
+  <label for="author">Author</label>
+  <select name="post[user_id]">
```
Comment (mthwtkr) [inline ashwin_vidiyala/Rails/layouts_assignment/app/views/posts/index.html.erb:29]: Very neat putting the users in the platform as the potential author of the post.

File: ashwin_vidiyala/Rails/app/views/students/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+    <h2>Cohort</h2>
+    <table>
+      <tr>
+        <th>First Name</th>
+        <th>Last Name</th>
+        <th>Email</th>
+      </tr>
+      <% @students.each do |student| %>
+        <% if student != @student %>
```
Comment (mthwtkr) [inline ashwin_vidiyala/Rails/app/views/students/show.html.erb:18]: Not lazy enough ;) Try to implement a method that does this for you.

**Comment by mthwtkr** [comment]
Everything else is looking good.

"Everything else is kind of simple" - Ashwin

Not everyone thinks it's simple ;)

File: ashwin_vidiyala/Rails/layouts_assignment/app/views/posts/index.html.erb (hunk 1 above)
Comment (ashwinvidiyala) [reply ashwin_vidiyala/Rails/layouts_assignment/app/views/posts/index.html.erb:29]: :)

File: ashwin_vidiyala/Rails/app/views/students/show.html.erb (hunk 2 above)
Comment (ashwinvidiyala) [reply ashwin_vidiyala/Rails/app/views/students/show.html.erb:18]: Haha yes sir. I have that method working now :)

**Comment by ashwinvidiyala** [comment]
Well maybe not everything else, but for the most part it was. When it came to the students' stuff though, it got slightly confusing, especially because I had to deal with the nested routes. 
But yeah, overall, I think all of this is making sense :)

**Pull request merged**